exercises/phone-number/src/example.c

```diff
@@ -29,11 +27,15 @@ char *phone_number_clean(const char *input)
    size_t j = 0;
 
    for (size_t i = 0; i < strlen(input); i++) {
-      if (isdigit(input[i])) {
+
+      if (isdigit(input[i]) || (i == 0 && input[0] == '+')) {
+
          if (j > VALID_NUMBER_LENGTH + 1) {
             break;
          }
-         output[j++] = input[i];
+         if (isdigit(input[i]))
+            output[j++] = input[i];
+
       } else if (strchr(VALID_NON_DIGIT_CHARS, input[i]) == NULL) {
```

Comment:
I know this isn't changed already by this PR, but, for safety's sake, should this condition add  `&& !isspace(input[i])` instead of including `" "` in the `VALID_NON_DIGIT_CHARS` string?

Reply:
Upon reflection, maybe leave this alone as it is not related to the PR

Reply:
So leave it as is then?

Reply:
Yep, apologies for the confusion 😁